src/main/webapp/script.js

```diff
@@ -1,3 +1,42 @@
+class ParameterField extends HTMLElement {
```

Comment:
Could you explain what this is/does?
Maybe also add a comment clarifying

Reply:
I add documentation to this file the the PR Delete Parameters, but this class is used as a template for each parameter input field for tags, ingredients, and steps. It makes it easy to add any of each, as opposed to manually adding a new HTML label, textarea, and button each time.

Reply:
I didn't know you could do that, interesting.

Reply:
+1.